spec/fixtures/cleanup-foreman-user-groups/foreman-answers-after.yaml

```
@@ -0,0 +1,2 @@
+foreman:
+  user_groups: []
```

```suggestion
foreman:
```

Should we add a 2nd piece of logic to the migration to remove the key entirely if the value ends up `[]`?

---

I prefer not to try to predict that situation and just do the targeted cleanup this is aimed at.

---

Actually, I suppose it would come back anyway after the installer run, because `[]` is the default in puppet-foreman